moderntribe/square-one
# Feature/docker improvements

## What does this do/fix?

- Fixes composer memory limit issues on larger projects
- Removes some outdated documentation
- Fixes the path to msmtp so the docker image can send emails to mailhog again
- Updates to the latest squareone docker 74-2.1.1 image


## QA

run a `so composer update` on a project based on this, should no longer time out with memory issues. 

```
Fatal error: Allowed memory size of 1610612736 bytes exhausted (tried to allocate 4096 bytes) in phar:///usr/local/bin/composer/src/Composer/DependencyResolver/Solver.php on line 223

Check https://getcomposer.org/doc/articles/troubleshooting.md#memory-limit-errors for more info on how to handle out of memory errors.Done.
```
Use a service in Wordpress that sends email, it should now appear in mailhog.

## Tests

Does this have tests?

- [x ] No, this doesn't need tests because this is all environment based.
